plugins/samson_ledger/config/initializers/samson_ledger/client.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+#
+# basic ledger event client
+#
+require 'faraday'
+require 'openssl'
+
+module Ledger
+  class Client
+    LEDGER_URI = "/api/v1/events"
+    class << self
+      def verify_env
+        raise "LEDGER_TOKEN is not set in ENV" unless ENV.fetch("LEDGER_TOKEN")
```

Comment:
... also don't need that since plugin_enabled? already takes care of checking ...